linux_os/guide/system/logging/journald/journald_compress/tests/correct_value.pass.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+# platform = multi_platform_all
+JOURNALD_CONFIG=/etc/systemd/journald.conf
```

Comment:
When the rule is templated we prefer templated test scenarios.

Reply:
Looks like there is not any templated tests after looking at https://github.com/ComplianceAsCode/content/tree/master/shared/templates/shell_lineinfile.